grouper/models.py

```diff
@@ -253,6 +253,10 @@ def name(self):
     def type(self):
         return "User"
 
+    @property
+    def is_enabled(self):
+        return self.enabled
```

Comment:
If we want to do that in the future, you rename `enabled` to `_enabled`, and make `enabled` a property.

Since right now we have a bunch of existing code referencing `enabled` (I assume), we end up with two different ways of checking the same thing that can get out of sync.